readme.txt

```diff
@@ -79,6 +79,25 @@ By default, the WebP Uploads module will only generate WebP versions of the imag
 
 == Changelog ==
 
+= 2.8.0 =
+
+**Features**
+
+* Infrastructure: Introduce UI for managing Performance Lab standalone plugins. ([864](https://github.com/WordPress/performance/pull/864))
+
+**Enhancements**
+
+* Infrastructure: Add support for plugin live preview in the plugin directory. ([890](https://github.com/WordPress/performance/pull/890))
+* Infrastructure: Allow module `can-load.php` callbacks to return a `WP_Error` with more information. ([891](https://github.com/WordPress/performance/pull/891))
+* Infrastructure: Implement admin pointer to indicate to the user they need to migrate modules to their standalone plugins. ([910](https://github.com/WordPress/performance/pull/910))
+* Infrastructure: Implement migration logic and UI from Performance Lab modules to their standalone plugins. ([899](https://github.com/WordPress/performance/pull/899))
+* Infrastructure: Reset admin pointer dismissal for module migration when the user activates a module. ([915](https://github.com/WordPress/performance/pull/915))
+
+**Bug Fixes**
+
+* Infrastructure: Fix construction of translation strings in admin/plugins.php. ([925](https://github.com/WordPress/performance/pull/925))
+* Infrastructure: Prevent sending `Location` header during `test_perflab_aea_clean_aea_audit_action`. ([926](https://github.com/WordPress/performance/pull/926))
```

Comment:
Actually, I see that this PR was merged into `trunk`: https://github.com/WordPress/performance/pull/926

Was it cherry-picked into `2.8.0`?

Reply:
This is my only question as well. Probably best to just remove this Readme line for now and ship that in 2.9.0?

Reply:
Updated in https://github.com/WordPress/performance/pull/941/commits/c68ca520761fdbc1a4d7a94ecea14e5d9228daf7